service/user_service.go

```diff
@@ -10,10 +10,12 @@ import (
 
 var ErrInvalidEmailPassword = errors.New("invalid email or password")
 
+var NoUsersExist = errors.New("No users exist at present")
```

Comment:
make new folders(pkg) errors/errors.go & add this errors in that file & then access